packages/server-wallet/e2e-test/pong/controller.ts

```diff
@@ -0,0 +1,45 @@
+import {Message} from '@statechannels/wire-format';
+import {SignedState} from '@statechannels/wallet-core';
+
+import {Wallet} from '../../src/wallet';
+
+export default class PongController {
+  private readonly wallet: Wallet = new Wallet();
+
+  public async acceptMessageAndReturnReplies(message: Message): Promise<Message> {
+    const {
+      recipient: to,
+      sender: from,
+      data: {signedStates},
+    } = message;
+
+    // FIXME: server-wallet is using wallet-core, not wire-format for
+    // types of messages between parties. e2e-test uses wire-format
```

Comment:
So, the wallet should call `serializeMessage` on outgoing messages, right?

Make an issue out of this comment?